src/GovUk.Education.ExploreEducationStatistics.Common/Services/ImportStatusService.cs

```diff
@@ -13,28 +14,46 @@ public enum IStatus
         UPLOADING,
         QUEUED,
         PROCESSING_ARCHIVE_FILE,
-        RUNNING_PHASE_1,
-        RUNNING_PHASE_2,
-        RUNNING_PHASE_3,
+        STAGE_1, // Basic row validation
+        STAGE_2, // Create locations and filters
+        STAGE_3, // Split Files
+        STAGE_4, // Split Files complete
```

Comment:
I think we can but where we did a check on `STAGE_4`, we need to check instead the underlying `DatafileImport. PercentageComplete` value being 100% rather than the weighted `ImportStatus.PercentageComplete`.

`ImportStatus.PercentageComplete` is only 30% at the end of `STAGE_3`.

To check the underlying `DatafileImport` value I added it as new field `PhasePercentageComplete` to ImportStatus.

Reply:
Nice addition, could be useful to show more fine-grained progress information in the future as well :+1: